vms/platformvm/config/execution_config_test.go

```diff
@@ -105,64 +102,14 @@ func TestExecutionConfigUnmarshal(t *testing.T) {
 			ChecksumsEnabled:             true,
 			MempoolPruneFrequency:        time.Minute,
 		}
-		require.Equal(expected, ec)
-	})
+		require.NoError(VerifyInitializedStruct(*expected))
+		require.NoError(VerifyInitializedStruct(expected.Network))
 
-	t.Run("default values applied correctly", func(t *testing.T) {
```

Comment:
This test is not really doing anything more than the "mix default and extracted values from json" test earlier